doc/cephfs/multimds.rst

```diff
@@ -36,7 +36,7 @@ Before ``max_mds`` can be increased, the ``allow_multimds`` flag must be set.
 The following command sets this flag for a filesystem instance.
 
 ::
```

Comment:
Looks good. But put a new line after `::` to make a box. 

See: http://docs.ceph.com/docs/master/cephfs/multimds/